pyccel/codegen/printing/fcode.py

```diff
@@ -535,7 +538,7 @@ def _print_AnnotatedComment(self, expr):
 
     def _print_Tuple(self, expr):
         import numpy
-        shape = numpy.shape(expr)
+        shape = list(reversed(numpy.shape(expr)))
```

Comment:
So you are reversing the shape here?

Reply:
I am reversing the shape so it matches the fortran indexing